site/docs/v1/tech/identity-providers/apple.adoc

```diff
@@ -95,7 +95,7 @@ On the next panel select the previously created App ID and click Save. Click Con
 
 == Import the Private Key
 
-Now that we have completed the Apple configuration, we can import the private key we collected in the previous step. To import the private key, navigate to [breadcrumb]#Settings -> Key Master# and select [breadcrumb]#Import Private Key# from the actions in the top right.
+Now that we have completed the Apple configuration, we can import the private key we collected in the previous step. To import the private key, navigate to [breadcrumb]#Settings -> Key Master# and select [breadcrumb]#Import Elliptic Private Key# from the actions in the top right.
```

Comment:
I would say this a little different in case Apple changes something in the future. Something like:

> Now that we have completed the Application configuration, we can import the private key we collected in the previous step. Unless otherwise directed by Apple, you can assume the provided private key is using the Elliptic curve. To import the private key, navigate to [breadcrumb]#Settings -> Key Master# and select [breadcrumb]#Import Elliptic Private Key# from the actions in the top right. Copy the provided Key identifier and key text into the corresponding fields.

Also, I don't know but it looks like from the screen shot that the user needs to find the provided key identifier and the key text from Apple and use it on this panel, is that correct?

Reply:
Yes, that makes sense. 

>   Copy the provided Key identifier and key text into the corresponding fields. Using a text editor, open the key file you downloaded in the previous step and copy the value into the Private key field.

I think that this GH PR cut off similar instructions already included in the doc.